services/beaconchain/src/main/java/tech/pegasys/artemis/services/beaconchain/BeaconChainController.java

```diff
@@ -180,6 +182,13 @@ public void initP2PNetwork() {
     }
   }
 
+  public void initRestAPI() {
+    STDOUT.log(Level.DEBUG, "BeaconChainController.initRestAPI()");
+    BeaconRestApi restApi =
+        new BeaconRestApi(chainStorageClient, p2pNetwork, config.getBeaconRestAPIPortNumber());
+    restApi.start();
```

Comment:
We probably should be stopping the rest API again in `BeaconChainController.stop()`

Reply:
done.